components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ApplicationBasicInfo.java

```diff
@@ -57,74 +58,142 @@ public void setApplicationId(int applicationId) {
     }
 
     /**
-     * @return
+     * Get application name.
+     *
+     * @return application name
      */
     public String getApplicationName() {
 
         return applicationName;
     }
 
     /**
-     * @param applicationName
+     * Set application name
+     *
+     * @param applicationName application name
      */
     public void setApplicationName(String applicationName) {
 
         this.applicationName = applicationName;
     }
 
     /**
-     * @return
+     * Get application description
+     *
+     * @return description
      */
     public String getDescription() {
 
         return description;
     }
 
     /**
-     * @param description
+     * Set application description
+     *
+     * @param description description
      */
     public void setDescription(String description) {
 
         this.description = description;
     }
 
+    /**
+     * Get application resource id
+     *
+     * @return application resource id
+     */
     public String getApplicationResourceId() {
 
         return applicationResourceId;
     }
 
+    /**
+     * Set application resource id
+     *
+     * @param applicationResourceId application resource id
+     */
     public void setApplicationResourceId(String applicationResourceId) {
 
         this.applicationResourceId = applicationResourceId;
     }
 
+    /**
+     * Get image URL
+     *
+     * @return image URL
+     */
     public String getImageUrl() {
 
         return imageUrl;
     }
 
+    /**
+     * Set application image URL
+     *
+     * @param imageUrl application image URL
+     */
     public void setImageUrl(String imageUrl) {
 
         this.imageUrl = imageUrl;
     }
 
+    /**
+     * Get access URL
+     *
+     * @return access URL
+     */
     public String getAccessUrl() {
 
         return accessUrl;
     }
 
+    /**
+     * Set access URL
+     *
+     * @param accessUrl access URL
+     */
     public void setAccessUrl(String accessUrl) {
 
         this.accessUrl = accessUrl;
     }
 
+    /**
+     * Get app owner details
+     *
+     * @return app owner details
+     */
     public User getAppOwner() {
 
         return appOwner;
     }
 
+    /**
+     * Set application owner details
+     *
+     * @param appOwner application owner details
+     */
     public void setAppOwner(User appOwner) {
 
         this.appOwner = appOwner;
     }
+
+    /**
+     * Get inbound auth key
+     *
+     * @return inbound auth key
+     */
+    public String getInboundKey() {
+
+        return inboundKey;
+    }
+
+    /**
+     * Set inbound auth key
```

Comment:
```suggestion
     * Set inbound auth key.
```
Each comment and param description should start with a capital letter and ends with a full stop. Check other places as well.